### ISSUE TITLE
Add learning rate scheduler

### ISSUE THREAD

**Issue by adambarla** [adambarla/NMT-augment#13]

**Comment by adambarla** [comment]
sorry i wasn't explicit. this is an example what i meant by scheduler like this:

https://pytorch.org/docs/stable/generated/torch.optim.lr_scheduler.ExponentialLR.html
